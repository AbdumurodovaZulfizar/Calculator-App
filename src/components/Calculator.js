import React from 'react';
import '../index.css';
import App from './App';

const Calculator = () => (
  <div>
    <h2>Calculator App</h2>
    <div className="col-6 mx-auto">
      <App />
    </div>
  </div>
);

export default Calculator;
