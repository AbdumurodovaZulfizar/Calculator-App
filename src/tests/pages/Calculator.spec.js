import React from 'react';
import renderer from 'react-test-renderer';
import Calculator from '../../components/Calculator';

test('should render correctly Calculator Page', () => {
  const component = renderer.create(<Calculator />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
