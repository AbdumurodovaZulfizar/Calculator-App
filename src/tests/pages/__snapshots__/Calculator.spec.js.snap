// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly Calculator Page 1`] = `
<div>
  <h2>
    Calculator App
  </h2>
  <div
    className="col-6 mx-auto"
  >
    <div
      data-testid="display"
    >
      <h5
        className="previous m-0 font"
      >
        0
        
      </h5>
    </div>
    <div
      className="buttonPanel row p-0 m-0"
      data-testid="panel"
    >
      <div
        className="ordinaries col-9 p-0"
      >
        <button
          className="ordinary"
          onClick={[Function]}
          type="button"
        >
          <h3
            className="color"
          >
            AC
          </h3>
        </button>
        <button
          className="ordinary"
          onClick={[Function]}
          type="button"
        >
          <h3
            className="color"
          >
            +/-
          </h3>
        </button>
        <button
          className="ordinary"
          onClick={[Function]}
          type="button"
        >
          <h3
            className="color"
          >
            %
          </h3>
        </button>
        <button
          className="ordinary"
          onClick={[Function]}
          type="button"
        >
          <h3
            className="color"
          >
            7
          </h3>
        </button>
        <button
          className="ordinary"
          onClick={[Function]}
          type="button"
        >
          <h3
            className="color"
          >
            8
          </h3>
        </button>
        <button
          className="ordinary"
          onClick={[Function]}
          type="button"
        >
          <h3
            className="color"
          >
            9
          </h3>
        </button>
        <button
          className="ordinary"
          onClick={[Function]}
          type="button"
        >
          <h3
            className="color"
          >
            4
          </h3>
        </button>
        <button
          className="ordinary"
          onClick={[Function]}
          type="button"
        >
          <h3
            className="color"
          >
            5
          </h3>
        </button>
        <button
          className="ordinary"
          onClick={[Function]}
          type="button"
        >
          <h3
            className="color"
          >
            6
          </h3>
        </button>
        <button
          className="ordinary"
          onClick={[Function]}
          type="button"
        >
          <h3
            className="color"
          >
            1
          </h3>
        </button>
        <button
          className="ordinary"
          onClick={[Function]}
          type="button"
        >
          <h3
            className="color"
          >
            2
          </h3>
        </button>
        <button
          className="ordinary"
          onClick={[Function]}
          type="button"
        >
          <h3
            className="color"
          >
            3
          </h3>
        </button>
        <button
          className="width-half"
          onClick={[Function]}
          type="button"
        >
          <h3
            className="color"
          >
            0
          </h3>
        </button>
        <button
          className="ordinary"
          onClick={[Function]}
          type="button"
        >
          <h3
            className="color"
          >
            .
          </h3>
        </button>
      </div>
      <div
        className="oranges col-3 p-0"
      >
        <button
          className="orange"
          onClick={[Function]}
          type="button"
        >
          <h3
            className=""
          >
            /
          </h3>
        </button>
        <button
          className="orange"
          onClick={[Function]}
          type="button"
        >
          <h3
            className=""
          >
            x
          </h3>
        </button>
        <button
          className="orange"
          onClick={[Function]}
          type="button"
        >
          <h3
            className=""
          >
            -
          </h3>
        </button>
        <button
          className="orange"
          onClick={[Function]}
          type="button"
        >
          <h3
            className=""
          >
            +
          </h3>
        </button>
        <button
          className="orange"
          onClick={[Function]}
          type="button"
        >
          <h3
            className=""
          >
            =
          </h3>
        </button>
      </div>
    </div>
  </div>
</div>
`;
